ext/drx/drx.h

```diff
@@ -132,8 +132,9 @@ drx_aflags_are_dead(instr_t *where);
 
 /** Flags for \p drx_insert_counter_update */
 enum {
-    DRX_COUNTER_64BIT = 0x01, /**< 64-bit counter is used for update. */
-    DRX_COUNTER_LOCK = 0x10,  /**< Counter update is atomic. */
+    DRX_COUNTER_64BIT = 0x01,    /**< 64-bit counter is used for update. */
+    DRX_COUNTER_LOCK = 0x10,     /**< Counter update is atomic. */
+    DRX_COUNTER_REL_ACQ = 0x100, /**< Release-acquire semantics for counter update. */
```

Comment:
Hmm, looks like DRX_COUNTER_LOCK was added with a mistaken not-next-bit value.  Best to leave it for compatibilty, but may as well take 0x02 for the new one.

Reply:
Yeah it was weird but I thought I should follow the convention. I changed the new flag to 0x02.